bittensor/utils/registration.py

```diff
@@ -469,7 +469,7 @@ class RegistrationStatistics:
     hash_rate: float
     difficulty: int
     block_number: int
-    block_hash: bytes
+    block_hash: str
```

Comment:
with going from `bytes` to `str`, lets make sure the E2E tests run and pass.

Reply:
Left the TODO